action.yml

```diff
@@ -5,8 +5,8 @@ inputs:
     description: 'TYPE of the new value to be updated'
     required: true
     default: 'IMAGE_TAG'
-  container-name:
-    description: 'Name of the container'
+  container-names:
```

Comment:
This will need to stay as container-name to avoid having to change it everywhere